skins/oasis/js/search-tracking/package.json

```diff
@@ -2,6 +2,6 @@
   "name": "wikia-search-tracking",
   "scripts": {},
   "dependencies": {
-    "@wikia/search-tracking": "github:Wikia/search-tracking#1.2.1"
```

Comment:
Use version tag after https://github.com/Wikia/search-tracking/pull/10 is released